File: exam_2020_march/ChangeBureau.java
import java.util.Scanner;

public class ChangeBureau 
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);

		int bitCoins = Integer.parseInt(sc.nextLine());
		double yuans = Double.parseDouble(sc.nextLine());
		double commiss = Double.parseDouble(sc.nextLine());
		
		double levs = bitCoins * 1168 + yuans * 0.15 * 1.76;
		double euros = levs / 1.95;
		double result = euros - ( euros * ( commiss / 100.0 ) );
		
		System.out.printf("%.2f", result);
    }
}

File: exam_2020_march/ChangeBureau.cpp
#include <iostream>

using namespace std;

int main()
{
	int bitCoins;
	cin >> bitCoins;
	
	double yuans;
	cin >> yuans;
	
	double commiss;
	cin >> commiss;
	
	double levs = bitCoins * 1168 + yuans * 0.15 * 1.76;
	double euros = levs / 1.95;
	double result = euros - ( euros * ( commiss / 100.0 ) );
	
	cout.setf(ios::fixed);
	cout.precision(2);
	cout << result;
}